test/integration/osl_hugo/controls/osl_hugo.rb

```diff
@@ -0,0 +1,13 @@
+control 'osl_hugo' do
+  describe directory('/opt/hugo-0.120.4') do
+    it { should exist }
+  end
+  describe directory('/opt/hugo') do
+    it { should exist }
+    its('link_path') { should cmp '/opt/hugo-0.120.4' }
+  end
```

Comment:
Let's replace these tests with a command test with `hugo version` returning the version you expect with an exit status of 0